databroker/v1.py

```diff
@@ -1608,38 +1608,58 @@ def _get_mongo_client(host, port):
     return client
 
 
-class _GetDocumentsRouter(event_model.DocumentRouter):
-    "This is used by Broker.get_documents."
+class _GetDocumentsRouter:
```

Comment:
promote this to event_model?

Reply:
Good idea. https://github.com/bluesky/event-model/issues/106